paper-api/src/main/java/org/bukkit/entity/Player.java

```diff
@@ -3477,6 +3478,18 @@ default void setNoTickViewDistance(int viewDistance) {
      */
     public void openSign(@NotNull Sign sign, @NotNull Side side);
 
+    /**
+     * Open a sign for editing by the player.
+     * <p>
+     * The sign must only be placed locally for the player, which can be done with {@link #sendBlockChange(Location, BlockData)} and {@link #sendBlockUpdate(Location, TileState)}.
+     * A side-effect of this is that no events will be called for this action.
+     * </p>
+     *
+     * @param block The block where the client has a sign placed
+     * @param side The side to edit
+     */
+    void openLocalSign(@NotNull BlockPosition block, @NotNull Side side);
```

Comment:
is local the right name here? do we use that naming elsewhere? maybe virtual? 

and I think the event that is used for the change should be mentioned here